crates/sp-domains-fraud-proof/src/lib.rs

```diff
@@ -156,6 +158,9 @@ pub enum FraudProofVerificationInfoResponse {
     BlockRandomness(Randomness),
     /// Encoded domain timestamp extrinsic using the timestamp from consensus state at a specific block hash.
     DomainTimestampExtrinsic(Vec<u8>),
+    /// Encoded domain transaction byte fee extrinsic using the storage price from consensus state at a
```

Comment:
Do you mean to say `Encoded consensus transaction byte fee extrinsic`?

Reply:
Yeah, the comment is outdated and forgot to update it.